app/_layouts/docs-v2.html

```diff
@@ -8,66 +8,6 @@
-
-        <div class="header-actions">
-          <div class="search-version-row">
-            {% unless page.no_version == true %}
-            <div class="versions-dropdown dropdown {% unless page.no_search == true %}no-right-border{% endunless %}">
-              <button class="page-header-btn" id="version-dropdown" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                Version {{page.kong_version}}
-                {% if page.kong_version == page.kong_latest.release %} <em>(latest)</em>{% endif %}
-                <span class="caret"></span>
-              </button>
-              <ul class="dropdown-menu dropdown-menu-right" role="menu" aria-labelledby="version-dropdown">
-                {% for ver in page.kong_versions reversed %}
-                {% assign version_page_url = page.url | replace: page.kong_version, ver.release %}
-                {% capture version_page_exists %}{% page_exists {{version_page_url}} %}{% endcapture %}<li {% if page.kong_version==ver.release %} class="active" {% endif %}>
-                <li {% if page.kong_version==ver.release %} class="active" {% endif %}>
-                  <a href="{% if version_page_exists == 'true' %}{{version_page_url}}{% else %}/{% if page.edition == 'gateway-oss' %}gateway-oss/{% endif %}{% if page.edition == 'enterprise' %}enterprise/{% endif %}{% if page.edition == 'mesh' %}mesh/{% endif %}{{ver.release}}{% endif %}"
-                    {% if ver.release==page.kong_version %} class="active" {% endif %}
-                    data-version-id="{{ver.release}}"
-                  >
-                  {{ver.release}}
-                  {% if ver.release == page.kong_latest.release %} <em>(latest)</em>{% endif %}
-                  </a>
-                </li>
-                {% endfor %}
-              </ul>
-            </div>
-            {% endunless %}
-            {% unless page.no_search == true %}
-            <div class="dropdown" id="searchBar">
-              <div id="getkong-algolia-search" class="search-input v2">
-                <img class="icon" src="/assets/images/icons/icn-magnifying-glass.svg" alt="search">
-                <input type="text" placeholder="Type to search" id="getkong-algolia-search-input" />
-              </div>
-            </div>
-          </div>
-          <img id="search-version-icon" src="/assets/images/icons/icn-magnifying-glass.svg" alt="search">
-          {% endunless %}
-          <div class="edit-this-page v2">
-            <a target="_blank" href="https://github.com/Kong/docs.konghq.com/tree/master/app/{{page.path}}">
-              <img src="/assets/images/logos/logo-github.svg" alt="github"/>
-            </a>
-          </div>
-        </div>
-      </nav>
-
-      <div class="title-version-search-container">
-        <div class="page-header-title">
-          <h1>{{page.title | flatify }}</h1>
```

Comment:
The layout is currently missing the page title and subtitle. Please add them right before `read_time` (lines 59-61). Something like:
```
          <h1>{{page.title}}</h1>
          {% if page.subtitle %}
          <h3 class="subtitle">{{page.subtitle}}</h3>
          {% endif %}
```